src/Microsoft.Framework.Logging/LoggerFactory.cs

```diff
@@ -15,6 +15,7 @@ public class LoggerFactory : ILoggerFactory
         private readonly Dictionary<string, Logger> _loggers = new Dictionary<string, Logger>(StringComparer.Ordinal);
         private ILoggerProvider[] _providers = new ILoggerProvider[0];
         private readonly object _sync = new object();
+        private LogLevel? _minimumLevel;
```

Comment:
no. The whole idea behind this bug is to prevent messages logged at the Debug level from being distributed to all the loggers by default.